server/src/pages/homepage/homepage-header.js

```diff
@@ -16,9 +16,8 @@ exports.HomePageHeader = class HomePageHeader extends React.Component {
 
   renderFxASignIn() {
       return (
-        this.props.isFirefox && this.props.isOwner ?
-          <SignInButton isAuthenticated={this.props.hasFxa} initialPage=""
-                        staticLink={this.props.staticLink} /> : null
+          <SignInButton isFxaAuthenticated={this.props.hasFxa} initialPage=""
```

Comment:
Renaming isAuthenticated SignInButton property to isFxAuthenticated to indicate its specific to Firefox Account and indicates accountId cookie exists after successful Fxa login